spec/services/articles/feeds/basic_spec.rb

```diff
@@ -9,7 +9,7 @@
   let!(:low_scoring_article) { create(:article, score: -1000) }
   let!(:month_old_story) { create(:article, published_at: 1.month.ago) }
 
-  it "returns articles in approximately published order" do
+  xit "returns articles in approximately published order" do
```

Comment:
is a fix in progress? if so, i think it's fine as is. but if not, we might want to put a `TODO` comment in here since we very intentionally are skipping it for a specific reason